src/MetaCompilation.Analyzers/UnitTests/UnitTests.cs

```diff
@@ -6,6 +6,9 @@
 using Test.Utilities;
 using Xunit;
 
+/*
```

Comment:
for these, it might be just easier to add #if false and #endif at the beginning and end of the file. and that will be easier to fix later and see history since content didn't change.